src/Pages/Auth/Register.jsx

```diff
@@ -93,66 +146,110 @@ const Register = () => {
         </div>
       </div>
       <div className="reg-form">
-        <form onSubmit={handleRegSubmit}>
+        <form onSubmit={formik.handleSubmit}>
+          {!registerData ? <h3>Email is taken</h3> : null}
```

Comment:
This is not the only error from the backend. You need to display any error from the backend here instead of using something static. You should be using state management to update the error message here

Reply:
Alright

Reply:
Ohk